meson.build

```diff
@@ -1194,13 +1195,10 @@ if features['egl-x11']
 endif
 
 plain_gl = get_option('plain-gl').require(
-    get_option('libmpv') and gl_allowed,
-    error_message: 'libmpv was not enabled!',
+    gl_allowed,
+    error_message: 'gl was not enabled!',
 )
-features += {'plain-gl': plain_gl.allowed()}
-if features['plain-gl']
-    features += {'gl': true}
-endif
+features += {'gl': plain_gl.allowed()}
```

Comment:
if plain_gl is disabled this will disable gl entirely

Reply:
Indeed, I'll drop this part